app/View/Label.php

```diff
@@ -96,6 +102,7 @@ public function render(callable $callback = null)
                         $title = str_replace('{COMPANY}', $asset->company->name, $settings->label2_title);
                         $settings->qr_text;
                         $assetData->put('title', $title);
+                        \Log::error("THE TITLE YOU ARE TRYING TO PUT IS: $title");
```

Comment:
Can we remove this and/or drop it down to debug or info level?

Reply:
yanked!